release-notes/portworx-enterprise.md

```diff
@@ -3,7 +3,9 @@ layout: page
 title: "PX-Enterprise Release notes"
 keywords: portworx, px-enterprise, release notes
 sidebar: home_sidebar
-redirect_from: "/px-enterprise-release-notes.html"
+redirect_from:
+  - /px-enterprise-release-notes.html
+  - /release-notes-1-0-4.html
```

Comment:
Should we also delete all the release notes in the root of the repo in favour of this?

Reply:
Okay let's ignore this for now